ci/common

```diff
@@ -46,16 +49,23 @@ setup_k3s () {
     }
     export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
 
-    setup_cni calico
+    setup_calico
 }
 
-setup_cni() {
-    # Install canal or calico as a CNI to enforce our NetworkPolicies. Note
-    # that canal was hard to get working with Travis CI for unknown reason.
+setup_calico() {
+    # Install calico as a CNI to enforce our NetworkPolicies.
+    #
+    # Note: Canal could have done the job as well, but we failed to set it up in
+    #       Travis CI. The await_dns_functionality_using_hub_pod function has a
+    #       note about this.
+    #
+    # Below we download the calico.yaml Kubernetes manifest and insert a
+    # container_settings section just below the phrase '"type": "calico"' and
+    # then `kubectl apply` it.
     #
     # ref: https://rancher.com/docs/k3s/latest/en/installation/network-options/
     #
-    curl -sfL https://docs.projectcalico.org/manifests/${1}.yaml \
+    curl -sfL https://docs.projectcalico.org/manifests/calico.yaml \
```

Comment:
```suggestion
    curl -sfL https://docs.projectcalico.org/v3.14/manifests/calico.yaml \
```
This will hopefully avoid unexpected breakages of our CI. Ideally this would include the patch version `v3.14.1` but they don't seem to publish those 😞

Reply:
Good idea, this is a good improvement though!